### ISSUE TITLE
Loss spiking around epoch 30-100

### ISSUE THREAD

**Issue by Yiming0131** [MIC-DKFZ/nnUNet#2790]
While training the Primus model, I observed a sharp increase in both training loss and validation loss between epochs 30 and 100. I would like to inquire whether this behavior could be attributed to the learning rate warm-up phase, or if it might indicate a deeper underlying issue. I would greatly appreciate any clarification you could provide.

![Image](https://github.com/user-attachments/assets/67b5d6f0-41ca-45a2-8d06-4ac523b4582c)

**Comment by TaWald** [comment]
Hi @Yiming0131,
Thanks for the pointer! It looks like the learning rate is too high for your dataset. I'd recommend decreasing it, for example, to 1e-4.

Out of curiosity, which dataset did you train this on? What kinds of segmented structures and which modalities are you working with?

I suspect this may be limited to the original Primus (Primus V1, if you will). That said, there has been quite a bit of additional development since then, and an updated PrimusV3 version is currently in the process of being released, which might be worth checking out.

Best,
Tassilo

**Comment by Karol-G** [comment]
Hey,

Is this issue still relevant? Otherwise, I will close it in 14 days.

Best,
Karol
